src/components/navbars/NavbarBottom.vue

```diff
@@ -0,0 +1,46 @@
+<script lang="ts" setup>
+import { Icon } from '@iconify/vue';
+import { useI18n } from 'vue-i18n';
+
+import messages from "../../i18n/index"
```

Comment:
When I set up the example I made `/i18n/index` have the keys for the Index-Route.
What do you think about storing the Strings under `/i18n/navbars` instead?

Reply:
Seems like a good idea. What do you think of the following structure (and is it even a good approach to have multiple .ts files and to have them in separate folders):
```
i18n
    index
        index.ts
        index.de.json
        index.en.json
        ...
    navbar
        navbar.ts
        navbar.de.json
        navbar.en.json
        ...
```

Reply:
yup. That looks good!

Reply:
Implemented it this exact way. Grouped the `index`, `navbar` and `notFound` translations into new separate folders.